docs/_getting-started-macos-ios.md

```diff
@@ -38,7 +38,7 @@ To check what is your current version of Ruby, you can run this command:
 ruby --version
 ```
 
-React Native uses [this version](https://github.com/facebook/react-native/blob/main/template/_ruby-version) of Ruby. You can also find which version your specific project needs in the `.ruby-version` file at root of your RN project.
+To use React Native your version of Ruby must meet the requirements defined in the [Gemfile](https://github.com/facebook/react-native/blob/main/Gemfile#L4).
```

Comment:
I will move this paragraph to **Ruby's Bundler** as the Gemfile is a configuration for `Bundler` itself